### PR TITLE
Update assertAuction to use ThresholdPrice from auctionInfo

### PR DESCRIPTION
**Pull request by MikeHathaway**
## Description

[use auctionInfo thresholdprice instead of recalculating in _assertAuction](https://github.com/ajna-finance/contracts/pull/1003/commits/9d4d182e7b2374a31314913c94930729e0421874)

## Tasks

- [ ] Changes to protocol contracts are covered by unit tests executed by CI.
- [ ] Protocol contract size limits have not been exceeded.
- [ ] Gas consumption for impacted transactions have been compared with the target branch, and nontrivial changes cited in the _Impact_ section above.
- [ ] Scope labels have been assigned as appropriate.
- [ ] Invariant tests have been manually executed as appropriate for the nature of the change.

### CODE REVIEW

File: tests/forge/unit/ERC20Pool/ERC20PoolLiquidationsDepositTake.t.sol [hunk 1]
```diff
@@ -151,7 +151,7 @@ contract ERC20PoolLiquidationsDepositTakeTest is ERC20HelperContract {
                 totalBondEscrowed: 0,
                 auctionPrice:      0,
                 debtInAuction:     0,
-                thresholdPrice:    9.607367579382098527 * 1e18,
+                thresholdPrice:    0,
```
Comment (EdNoepel) [inline tests/forge/unit/ERC20Pool/ERC20PoolLiquidationsDepositTake.t.sol:154]: Much better.

File: tests/forge/unit/ERC20Pool/ERC20PoolLiquidationsDepositTake.t.sol [hunk 2]
```diff
@@ -326,7 +326,7 @@ contract ERC20PoolLiquidationsDepositTakeTest is ERC20HelperContract {
                 totalBondEscrowed: 0.214827269923259784 * 1e18,
                 auctionPrice:      8.982038363413219840 * 1e18,
                 debtInAuction:     0.181853204762687052 * 1e18,
-                thresholdPrice:    0,
+                thresholdPrice:    9.607367579382098527 * 1e18,
```
Comment (EdNoepel) [inline tests/forge/unit/ERC20Pool/ERC20PoolLiquidationsDepositTake.t.sol:329]: Much better.

**Review by EdNoepel — APPROVED** [review]
LGTM 👌

**Review by ith-harvey — APPROVED** [review]
LGTM

**Pull request merged**